packages/components/generateFlowTypes.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+for i in $(find dist/* -name "*.d.ts"); do
+    [ -f "$i" ] || break
+    NAME=${i%%.*}
+    echo "Generating flowtype for $NAME"
+    node_modules/.bin/flowgen $i -o $NAME.flow.js
```

Comment:
does `npx flowgen $i -o $NAME.flow.js` work?

Reply:
yep - I'll change it to that